tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/truncate_fitbit_data_test.py

```diff
@@ -0,0 +1,178 @@
+        self.fq_dataset_name = '.'.join(fq_dataset_name[:-1])
+
+        super().setUp()
+
+    def test_truncate_data(self):
+        """
+        Tests that the specifications for the SANDBOX_QUERY and TRUNCATE_FITBIT_DATA_QUERY
+        perform as designed.
+
+        Validates pre conditions, tests execution, and post conditions based on the load
+        statements and the tables_and_counts variable.
+        """
+        queries = []
+        as_query = self.jinja_env.from_string("""
+            INSERT INTO `{{fq_dataset_name}}.{{fitbit_table}}`
+            (person_id, date)
+            VALUES
+            (111, date('2018-11-26')),
+            (222, date('2019-11-26')),
+            (333, date('2020-11-26')),
+            (444, date('2021-11-26'))""").render(
+            fq_dataset_name=self.fq_dataset_name,
+            fitbit_table=common.ACTIVITY_SUMMARY)
+        queries.append(as_query)
+
+        hr_query = self.jinja_env.from_string("""
+            INSERT INTO `{{fq_dataset_name}}.{{fitbit_table}}` 
+            (person_id, datetime)
+            VALUES
+            (111, (DATETIME '2018-11-26 00:00:00')),
+            (222, (DATETIME '2019-11-26 00:00:00')),
+            (333, (DATETIME '2020-11-26 00:00:00')),
+            (444, (DATETIME '2021-11-26 00:00:00'))""").render(
+            fq_dataset_name=self.fq_dataset_name,
+            fitbit_table=common.HEART_RATE_MINUTE_LEVEL)
+        queries.append(hr_query)
+
+        hrs_query = self.jinja_env.from_string("""
+            INSERT INTO `{{fq_dataset_name}}.{{fitbit_table}}`
+            (person_id, date)
+            VALUES
+            (111, date('2018-11-26')),
+            (222, date('2019-11-26')),
+            (333, date('2020-11-26')),
+            (444, date('2021-11-26'))""").render(
+            fq_dataset_name=self.fq_dataset_name,
+            fitbit_table=common.HEART_RATE_SUMMARY)
+        queries.append(hrs_query)
+
+        sid_query = self.jinja_env.from_string("""
+            INSERT INTO `{{fq_dataset_name}}.{{fitbit_table}}` 
+            (person_id, datetime)
+            VALUES
+            (111, (DATETIME '2018-11-26 00:00:00')),
+            (222, (DATETIME '2019-11-26 00:00:00')),
+            (333, (DATETIME '2020-11-26 00:00:00')),
+            (444, (DATETIME '2021-11-26 00:00:00'))""").render(
+            fq_dataset_name=self.fq_dataset_name,
+            fitbit_table=common.STEPS_INTRADAY)
+        queries.append(sid_query)
+
+        self.load_test_data(queries)
+
+        # Expected results list
+        tables_and_counts = [{
+            'fq_table_name':
+                '.'.join([self.fq_dataset_name, common.ACTIVITY_SUMMARY]),
+            'fq_sandbox_table_name':
+                self.fq_sandbox_table_names[0],
+            'fields': ['person_id', 'date'],
+            'loaded_ids': [111, 222, 333, 444],
+            'sandboxed_ids': [111],
+            'cleaned_values': [(222, parser.parse('2019-11-26').date()),
```

Comment:
I think the sandboxed and cleaned need to be swapped? From my understanding anything after 11/26/19 needs to be truncated? So 222, 333, 444 should be sandboxed and 111 remain?

Reply:
You're right. The wording in the ticket had me a little confused, I confirmed it with Heather 👍🏾